tests/unit_tests/connectors/test_offline_connector.py

```diff
@@ -0,0 +1,88 @@
+from apscheduler.schedulers.background import BackgroundScheduler
+from yggdrasil_engine.engine import UnleashEngine
+
+from tests.utilities.mocks.mock_features import MOCK_FEATURE_RESPONSE
+from UnleashClient.connectors import OfflineConnector
+from UnleashClient.constants import FEATURES_URL
+
+
+def test_offline_connector_load_features(cache_empty):
+    engine = UnleashEngine()
+    scheduler = BackgroundScheduler()
+    temp_cache = cache_empty
+
+    temp_cache.set(FEATURES_URL, MOCK_FEATURE_RESPONSE)
+
+    connector = OfflineConnector(
+        engine=engine,
+        cache=temp_cache,
+        scheduler=scheduler,
+    )
+
+    connector.load_features()
+
+    assert engine.is_enabled("testFlag", {})
+
+
+def test_offline_connector_load_features_empty_cache(cache_empty):
+    engine = UnleashEngine()
+    scheduler = BackgroundScheduler()
+    temp_cache = cache_empty
+
+    connector = OfflineConnector(
+        engine=engine,
+        cache=temp_cache,
+        scheduler=scheduler,
+    )
+
+    connector.load_features()
+
+    assert not engine.is_enabled("testFlag", {})
```

Comment:
This assert here tells us nothing, this will always pass since engine.is_enabled will return None here if there's no feature